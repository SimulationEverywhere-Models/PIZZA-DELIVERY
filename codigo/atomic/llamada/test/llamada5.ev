00:00:05:000 Lista_ll 1
00:00:10:000 Esperar 1
00:00:15:000 Llamada_in 1
00:00:15:005 Llamada_in 2
00:00:20:000 Libre 1
00:00:25:000 Parar_ll 1
