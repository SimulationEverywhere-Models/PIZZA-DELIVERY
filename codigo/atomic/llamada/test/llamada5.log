Mensaje I / 00:00:00:000 / Root(00) para top(01)
Mensaje I / 00:00:00:000 / top(01) para llamada(02)
Mensaje D / 00:00:00:000 / llamada(02) / ... para top(01)
Mensaje D / 00:00:00:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:05:000 / Root(00) / lista_ll /      1.00000 para top(01)
Mensaje X / 00:00:05:000 / top(01) / lista_ll /      1.00000 para llamada(02)
Mensaje D / 00:00:05:000 / llamada(02) / ... para top(01)
Mensaje D / 00:00:05:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:10:000 / Root(00) / esperar /      1.00000 para top(01)
Mensaje X / 00:00:10:000 / top(01) / esperar /      1.00000 para llamada(02)
Mensaje D / 00:00:10:000 / llamada(02) / ... para top(01)
Mensaje D / 00:00:10:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:15:000 / Root(00) / llamada_in /      1.00000 para top(01)
Mensaje X / 00:00:15:000 / top(01) / llamada_in /      1.00000 para llamada(02)
Mensaje D / 00:00:15:000 / llamada(02) / 00:00:00:015 para top(01)
Mensaje D / 00:00:15:000 / top(01) / 00:00:00:015 para Root(00)
Mensaje X / 00:00:15:005 / Root(00) / llamada_in /      2.00000 para top(01)
Mensaje X / 00:00:15:005 / top(01) / llamada_in /      2.00000 para llamada(02)
Mensaje D / 00:00:15:005 / llamada(02) / 00:00:00:000 para top(01)
Mensaje D / 00:00:15:005 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:15:005 / Root(00) para top(01)
Mensaje * / 00:00:15:005 / top(01) para llamada(02)
Mensaje Y / 00:00:15:005 / llamada(02) / ocupado /      1.00000 para top(01)
Mensaje D / 00:00:15:005 / llamada(02) / 00:00:00:010 para top(01)
Mensaje Y / 00:00:15:005 / top(01) / ocupado /      1.00000 para Root(00)
Mensaje D / 00:00:15:005 / top(01) / 00:00:00:010 para Root(00)
Mensaje * / 00:00:15:015 / Root(00) para top(01)
Mensaje * / 00:00:15:015 / top(01) para llamada(02)
Mensaje Y / 00:00:15:015 / llamada(02) / no_atiende /      1.00000 para top(01)
Mensaje D / 00:00:15:015 / llamada(02) / ... para top(01)
Mensaje Y / 00:00:15:015 / top(01) / no_atiende /      1.00000 para Root(00)
Mensaje D / 00:00:15:015 / top(01) / ... para Root(00)
Mensaje X / 00:00:20:000 / Root(00) / libre /      1.00000 para top(01)
Mensaje X / 00:00:20:000 / top(01) / libre /      1.00000 para llamada(02)
Mensaje D / 00:00:20:000 / llamada(02) / ... para top(01)
Mensaje D / 00:00:20:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:25:000 / Root(00) / parar_ll /      1.00000 para top(01)
Mensaje X / 00:00:25:000 / top(01) / parar_ll /      1.00000 para llamada(02)
Mensaje D / 00:00:25:000 / llamada(02) / ... para top(01)
Mensaje D / 00:00:25:000 / top(01) / ... para Root(00)
