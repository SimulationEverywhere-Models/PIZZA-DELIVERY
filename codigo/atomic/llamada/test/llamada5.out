00:00:15:005 ocupado 1
00:00:15:015 no_atiende 1
